src/HeaderBar.vala

```diff
@@ -39,7 +39,59 @@ public class Mail.HeaderBar : Gtk.HeaderBar {
         paned_start_grid = new Gtk.Grid ();
         paned_start_grid.add (compose_button);
 
+        var load_images_switch = new Gtk.Switch ();
+
+        var settings = new GLib.Settings ("io.elementary.mail");
+        settings.bind ("always-load-remote-images", load_images_switch, "active", SettingsBindFlags.DEFAULT);
+
+        var load_images_grid = new Gtk.Grid ();
+        load_images_grid.column_spacing = 12;
+        load_images_grid.margin_end = 6;
+        load_images_grid.margin_start = 6;
+        load_images_grid.add (new Gtk.Label (_("Always Show Remote Images")));
+        load_images_grid.add (load_images_switch);
+
+        var load_images_menuitem = new Gtk.Button ();
+        load_images_menuitem.get_style_context ().add_class (Gtk.STYLE_CLASS_MENUITEM);
+        load_images_menuitem.add (load_images_grid);
+
+        var account_settings_menuitem = new Gtk.ModelButton ();
```

Comment:
I don't think it's a problem, but why use a `ModelButton` instead of a `MenuItem` here?